bin/webpack.js

```diff
@@ -32,6 +32,13 @@
 		"info"
 	];
 
+	const nodeFlags = process.argv.filter((arg) => arg.includes("node."));
+
+	if (nodeFlags.length) {
+		require("./process-node-flags")(process.argv);
+		return;
```

Comment:
Why there's a `return` here?

Reply:
Same reason as used [here](https://github.com/webpack/webpack-cli/blob/master/bin/webpack.js#L46). If we don't exit, yargs will warn saying these flags are unknown and the normal cli flow will follow, with this return, the other file can strip those flags, spawn another webpack-cli process and work as intended.

Reply:
Tests added. Let me know if they aren't enough. I've tried to add as many flags as possible.

Reply:
Okay great, thanks!